### ISSUE TITLE
ksh `..` vs $(..) differences when results are massive

### ISSUE THREAD

**Issue by ColemanTom** [att/ast#33]
If you create a variable using backticks vs parenthesis which results in a large amount of output, the backticks version will cut the results off. For example:

```
a=`find . -type f`
b=$(find . -type f)

[ "$a" == "b" ]; echo $?
> 1

echo "$a" | wc -c
> 1388545
echo "$b" | wc -c
> 1881923
```

It appears the backticks method has a hard character limit of 1388545 (at least on the system I am testing this on).

**Comment by bitstreamout** [comment]
On Fri, Mar 24, 2017 at 03:25:00PM -0700, ColemanTom wrote:
> Closed #33.

Hmmm ... IMHO the bug is still there but upstream seems not be active anymore

-- 
  "Having a smoking section in a restaurant is like having
          a peeing section in a swimming pool." -- Edward Burr

**Comment by ColemanTom** [comment]
@bitstreamout I realised after I posted it I was actually in bash at the time when I got my numbers. I knew there was a bug in KSH also, but I couldn't remember if it was exactly the same or if it was something different. I thought I should close it until I can determine if it was the exact same bug, or a different one. Thanks for reminding me about this as I can check that today.
